waspc/data/Generator/templates/server/src/app.js

```diff
@@ -1,32 +1,19 @@
 import express from 'express'
-import cookieParser from 'cookie-parser'
-import logger from 'morgan'
-import cors from 'cors'
-import helmet from 'helmet'
 
 import HttpError from './core/HttpError.js'
 import indexRouter from './routes/index.js'
-import config from './config.js'
 
 // TODO: Consider extracting most of this logic into createApp(routes, path) function so that
 //   it can be used in unit tests to test each route individually.
 
 const app = express()
 
-app.use(helmet())
-app.use(cors({
-  // TODO: Consider allowing users to provide an ENV variable or function to further configure CORS setup.
-  origin: config.allowedCORSOrigins,
-}))
-app.use(logger('dev'))
-app.use(express.json())
-app.use(express.urlencoded({ extended: false }))
-app.use(cookieParser())
```

Comment:
What if some middleware we still installed here always, like logger and helmet and stuff, but then some other gets applied per-route basis? Because maybe for some of them it makes sense to use different rules?

Reply:
I tried to document our defaults a bit, and made the note that `express.json` was the main one we need for Wasp to work. Plus a warning :) Hopefully this will be enough for them on the global front.